### PR TITLE
Game state

### PR DESCRIPTION
**Pull request by IgorAleks88**
add game-state object

### CODE REVIEW

**Review by DenisOleksiuk — DISMISSED** [review]

File: src/js/components/mainMenu.js [hunk 1]
```diff
@@ -53,7 +53,10 @@ class Menu {
         if (validation(usersInfo) && usersInfo.names.length) {
           const intro = this.menu.parentElement.parentElement.parentElement;
           intro.classList.toggle('intro--hide');
-          initHotSeatGame('Player1', 'Player2'); //! Hardcoded for 2 players. Should take player names as arguments
+          console.log(usersInfo);
```
Comment (DenisOleksiuk) [inline src/js/components/mainMenu.js:56]: remove console.log from pull request

File: src/js/utility/getCardObject.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+export default function getCardObject(id, arr) {
```
Comment (DenisOleksiuk) [inline src/js/utility/getCardObject.js:1]: there should be a separate pull request for this task

**Review by Ferri0 — APPROVED** [review]

**Review by DenisOleksiuk — APPROVED** [review]
